vyper/parser/expr.py

```diff
@@ -147,9 +147,9 @@ def number(self):
             )
             if len(orignum[2:]) != total_bits:  # Support only full formed bit definitions.
                 raise InvalidLiteralException((
-                    "Bit notation requires a multiple of 8 bits / 1 byte. {} "
-                    "bit(s) are missing."
-                ).format(total_bits - len(orignum[2:])), self.expr)
+                    f"Bit notation requires a multiple of 8 bits / 1 byte. "
+                    f"{total_bits - len(orignum[2:])} bit(s) are missing."
```

Comment:
Combined with aboev and below
```suggestion
                    f"{total_bits - len(orignum[2:])} bit(s) are missing.",
```